.eslintrc

```diff
@@ -1,121 +1,10 @@
-  },
-  "globals": {
-    "pathfora": true,
-    "$": true,
-    "FB": true,
-    "gapi": true
-  },
-  "rules": {
-    "block-scoped-var": "error",
-    "camelcase": "error",
-    "comma-style": [
-      "error",
-      "last"
-    ],
-    "comma-spacing": [
-      "error",
-      {
-        "before": false,
-        "after": true,
-      }
-    ],
-    "curly": [
-      "error",
-      "all"
-    ],
-    "dot-notation": "error",
-    "eqeqeq": [
-      "error",
-      "allow-null"
-    ],
-    "guard-for-in": "error",
-    "new-cap": "error",
-    "no-caller": "error",
-    "no-cond-assign": "error",
-    "no-debugger": "error",
-    "no-empty": "error",
-    "no-eval": "error",
-    "no-extend-native": "error",
-    "no-irregular-whitespace": "error",
-    "no-iterator": "error",
-    "no-loop-func": "error",
-    "no-multi-str": "error",
-    "no-proto": "error",
-    "no-script-url": "error",
-    "no-sequences": "error",
-    "no-undef": "error",
-    "no-unused-vars": "error",
-    "no-with": "error",
-    "semi": [
-      "error",
-      "always"
-    ],
-    "valid-typeof": "error",
-    "wrap-iife": [
-      "error",
-      "outside"
-    ],
-    "space-infix-ops": "error",
-    "indent": [
-        "error",
-        2,
-        { "SwitchCase": 0, "VariableDeclarator": 2 },
-    ],
-    "comma-dangle": "error",
-    "no-dupe-args": "error",
-    "no-dupe-keys": "error",
-    "no-duplicate-case": "error",
-    "no-ex-assign": "error",
-    "no-extra-semi": "error",
-    "no-inner-declarations": "error",
-    "no-invalid-regexp": "error",
-    "no-obj-calls": "error",
-    "no-sparse-arrays": "error",
-    "no-unreachable": "error",
-    "use-isnan": "error",
-    "no-lone-blocks": "error",
-    "no-multi-spaces": "error",
-    "no-new-func": "error",
-    "no-new-wrappers": "error",
-    "no-redeclare": "error",
-    "no-useless-concat": "error",
-    "radix": "error",
-    "yoda": "error",
-    "no-delete-var": "error",
-    "no-shadow": "error",
-    "no-undef-init": "error",
-    "no-undefined": "error",
-    "no-use-before-define": "error",
-    "global-require": "error",
-    "array-bracket-spacing": [
-        "error",
-        "never"
-    ],
-    "block-spacing": "error",
-    "brace-style": "error",
-    "computed-property-spacing": "error",
-    "eol-last": "error",
-    "no-lonely-if": "error",
-    "no-multiple-empty-lines": "error",
-    "no-mixed-spaces-and-tabs": "error",
-    "no-trailing-spaces": "error",
-    "space-before-blocks": "error",
-    "space-before-function-paren": [
-        "error",
-        { "anonymous": "always", "named": "always" }
-    ],
-    "keyword-spacing": "error",
-    "space-in-parens": "error",
-    "quotes": [
-        "error",
-        "single",
-        { "avoidEscape": true }
-    ]
-  }
-}
\ No newline at end of file
+extends:
+- ./node_modules/lytics-js-styleguide/.eslintrc
+env:
+  browser: true
+  jasmine: true
+globals:
+  pathfora: true
+  "$": true
+  FB: true
+  gapi: true
```

Comment:
Would it be acceptable to add any pathfora-specific rules here? For example the enforcement of single quotes. Since it's already everywhere in the code it would be nice to maintain. I don't think we would plan on adding many rules back, but a few one-off things like that might be nice to keep.